common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -13218,6 +13226,15 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=412)
     public static final String A_zimbraPrefIdentityName = "zimbraPrefIdentityName";
 
+    /**
+     * whether end user choose to use IMAP feature. If it is set to TRUE,
+     * IMAP feature is available only when zimbraImapEnabled is TRUE.
+     *
+     * @since ZCS 9.0.0
```

Comment:
9.1.0

Reply:
Next release version is yet to be finalised. Keeping 9.0.0